src/store/socket.js

```diff
@@ -481,6 +485,41 @@ class LiveSession {
     }
   }
 
+  /**
+   * Publish a player pronouns update
+   * @param player
+   * @param value
+   */
+  sendPlayerPronouns({ player, value }) {
+    //send pronoun only for the seated player or storyteller
+    if (this._isSpectator && this._store.state.session.playerId !== player.id)
+      return;
+    const index = this._store.state.players.players.indexOf(player);
+    this._send("pronouns", { index, value, fromST: !this._isSpectator });
```

Comment:
Minor nitpick, but to save a bit of bandwidth, I'd suggest using an array instead of an object here:
```suggestion
    this._send("pronouns", [ index, value, !this._isSpectator ]);
```